torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -260,7 +260,8 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
       return runTorchSlice_opset9(node, inputTensorValues);
     } else if (
         opset_version == ONNX_OPSET_10 || opset_version == ONNX_OPSET_11 ||
-        opset_version == ONNX_OPSET_12 || opset_version == ONNX_OPSET_13) {
+        opset_version == ONNX_OPSET_12 || opset_version == ONNX_OPSET_13 ||
+        opset_version == ONNX_OPSET_14) {
```

Comment:
For each new ops version, seems like we always need to update 2 places: here and adding a new ops version, like ONNX_OPSET_xx, in constant_fold.h. How about optimizing the code so that we only need to update 1 file constant_fold.h in future:

1. Add a new static vector in constant.fold.h file. This vector will be initialized with all existing opsets, from ONNX_OPSET_11 to ONNX_OPSET_14.
2. In here, we just call find() method to check if opset_version exists in this vector, instead of having so many OR statements.

In future, if we have a new opset 15, we just need to add it into the static vector defined in constant_fold.h file. No more work needs to be done here.

Reply:
good point. I also think this condition is equivalent to `opset_version >= ONNX_OPSET_10`.